src/zenml/orchestrators/utils.py

```diff
@@ -61,11 +91,13 @@ def execute_step(
     Returns:
         Optional execution info returned by the launcher.
     """
-    step_name = tfx_launcher._pipeline_node.node_info.id  # type: ignore[attr-defined] # noqa
+    step_name = tfx_launcher._pipeline_node.node_info.id  # type: ignore[attr-defined]
     start_time = time.time()
     logger.info(f"Step `{step_name}` has started.")
     try:
         execution_info = tfx_launcher.launch()
+        if is_cached_step(execution_info):  # type: ignore [arg-type]
+            logger.info(f"Using cached version of `{step_name}` step.")
```

Comment:
Can we also add some more details? Like the actual function name instead? So e.g. if you pass in `importer_mnist` and the pipeline parameter name is `importer` then we can say: `f"Using cached version for {pipeline_step_name} [{step_name}] from run_id: {run_id}"`